<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QcMainWindow</class>
 <widget class="QMainWindow" name="QcMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qc</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../home/src/res.qrc">
    <normaloff>:/images/search.png</normaloff>:/images/search.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Start Date:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dtpStartDate">
        <property name="displayFormat">
         <string>dd-MMM-yyyy</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>End Date:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dtpEndDate">
        <property name="displayFormat">
         <string>dd-MMM-yyyy</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Search:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtSearch"/>
      </item>
      <item>
       <widget class="QPushButton" name="cmdShowOcs">
        <property name="text">
         <string>Show OCs</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="CustomQueryView" name="treeView"/>
    </item>
    <item>
     <widget class="QTreeWidget" name="trvQcData">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <column>
       <property name="text">
        <string>OC #</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Date</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Customer</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Product</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Carton Weight</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Order Qty</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Run</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomQueryView</class>
   <extends>QTreeView</extends>
   <header>customqueryview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../home/src/res.qrc"/>
 </resources>
 <connections/>
</ui>
